doc/changes/latest.rst

```diff
@@ -24,6 +24,7 @@ Enhancements
 
 - :bdg-dark:`Code` Add footer to masker reports (:gh:`4307` by `Rémi Gau`_).
 
+- :bdg-dark:`Code` Add a :meth:`~glm.first_level.FirstLevelModel.fit_array` method to first first level GLMs directly to 2D array (:gh:`4112` by TODO henrymj and `Rémi Gau`_).
```

Comment:
@henrymj 

I am going to need some info to list you as contributor

See this example

```
authors:

  - given-names: Dimitri Papadopoulos
    family-names: Orfanos
    website: https://github.com/DimitriPapadopoulos
    affiliation: NeuroSpin, C.E.A., Université Paris-Saclay, Gif-sur-Yvette, France
    orcid: https://orcid.org/0000-0002-1242-8990
```

Reply:
- given-names: Henry
    family-names: Jones
    website: https://github.com/henrymj
    affiliation: Institute for Mind and Biology, University of Chicago, Chicago, IL, United States of America
    orcid: https://orcid.org/0000-0001-7719-3646